tests/conftest.py

```diff
@@ -12,6 +12,10 @@
 
 from .helpers import Spawn
 
+pytest_plugins = [
+    "tests.templates",
+]
```

Comment:
I think the way tests are written is a bit more theoretical and complex than necessary.

To test the contents of ˋ_copier_conf.operationˋ, you could just do a single test that prints its output in a file. Then copy, recopy and update in sequence, and assert on each step the changes on the file.

In particular, you're not testing the use of this feature under ˋ_excludeˋ for updates, which is the specific use case we want to support. This could be a separate test with a separate template.

Since those would be just 2 tests, you wouldn't need to share these extra fixtures accross modules, and that'd make the tests easier to read and follow.

So would you please refactor the tests following this method?